sqrl-testing/sqrl-integration-tests/src/test/resources/usecases/duckdb/snapshots-queries/iceberg-query.snapshot

```diff
@@ -1 +1 @@
-{"data":{"MyTable":[{"id":101.0,"hello":"hello world"},{"id":99.0,"hello":"hello world"},{"id":98.0,"hello":"hello world"},{"id":97.0,"hello":"hello world"},{"id":96.0,"hello":"hello world"},{"id":95.0,"hello":"hello world"},{"id":95.0,"hello":"hello world"},{"id":94.0,"hello":"hello world"},{"id":93.0,"hello":"hello world"},{"id":92.0,"hello":"hello world"}]}}
\ No newline at end of file
+{"data":{"MyTable1":[{"id":101.0,"hello":"hello world"},{"id":29.0,"hello":"hello world"},{"id":85.0,"hello":"hello world"},{"id":14.0,"hello":"hello world"},{"id":85.0,"hello":"hello world"},{"id":35.0,"hello":"hello world"},{"id":95.0,"hello":"hello world"},{"id":30.0,"hello":"hello world"},{"id":34.0,"hello":"hello world"},{"id":58.0,"hello":"hello world"}]}}
```

Comment:
the data looks wrong

Reply:
looks like we lost the default order?

Reply:
does it? the previous queries had an order statement which made it 'look' more correct. ill double check that it is actually iceberg queries?

Reply:
Yeah, maybe the old result was wrong. I just noticed the difference, i.e. one of them must  be wrong ;-)

Reply:
Actually, this is the order the data is in.